### PR TITLE
Add pass_filenames parameter to the project.

### LINKED ISSUE #56: How to setup release to include config attributes
It seems that the script overwrites the yaml confile by default (instead of updating it). If I need to maintain a config param such as `pass_filenames: false`. How would I go about doing that?

The repo in question: https://github.com/adithyabsk/mirrors-brakeman

### PR DESCRIPTION
**Pull request by adithyabsk**
* Add pass_filenames to relevant files
  * template_file
  * argparser
  * unit tests

Closes #56

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,6 @@
 /.tox
 /dist
 /venv*
+.python-version
+.idea/*
+htmlcov/*
```
Comment (asottile) [inline .gitignore]: please don't touch gitignore, thanks

Comment (adithyabsk) [reply .gitignore:12]: fair enough, I guess I'll move stuff I want to ignore specific to my setup to the [`.git/info/exclude`](https://git-scm.com/docs/gitignore#_description) file.

**Comment by adithyabsk** [comment]
@asottile let me know if you have any other comments

**Comment by asottile** [comment]
sorry, I have to revert this -- it defaults to something which is not the default

**Comment by adithyabsk** [comment]
Is this because of a change to pre-commit?

**Comment by asottile** [comment]
no, this broke pre-commit/mirrors-eslint

the default is `pass_filenames: true`, this changes it in `pre-commit-mirror-maker` to `pass_filenames: false`

**Comment by asottile** [comment]
for clarity, this patch was incorrect to begin with but I didn't notice it.  it should default to `pass_filenames: true` and the option should be `--no-pass-filenames`

**Comment by adithyabsk** [comment]
Ah I see, it would be preferable to keep the behavior and support `--no-pass-filenames`. Then I'm not sure why this patch fixed my use case here if it was broken in the first place (defaulted to incorrect behavior) because I used multiple repos that made user of pre-commit-mirror-maker:

https://github.com/adithyabsk/rubystorefront/pull/10/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R27

**Comment by adithyabsk** [comment]
It's been a long time since I wrote that code and I'm not actively developing in Ruby so it's not mission-critical for me either way.

**Comment by asottile** [comment]
feel free to send a patch trying again if you'd like, but for now it's been reverted because it was a breaking change

**Pull request merged**